.circleci/deploy.sh

```diff
@@ -52,3 +52,15 @@ done
 
 # Upload examples
 s3cmd --no-mime-magic --guess-mime-type put -r ./examples/* $s3_example_bucket
+
+if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
Very subtle but I think adding comment between 55 and 56 may be good because of consistency and readability.

Reply:
Comment may be like this: `# Notify SkyWay team Slack of the new release` .